### DISCUSSION TITLE
How to stay logged in on multiple devices?

### DISCUSSION THREAD

**Discussion by dattito** [goauthentik/authentik#5054] [Q&A]
When I log in in authentik on one device, and then use the login with the same account on another device, the old one get's logged out. This is really annoying... How can I turn on staying logged in on multiple devices/sessions?

**Comment by j-streicher** [comment]
I think you can edit the login stage. There is a boolean option where you can stop terminating old session. It can cause that you will collect a lot of sessions until each session expires

**Comment by Zobozodiac** [comment]
Hi just adding to this, I'm the same where I immediately get logged out of a device if I log in on another. I think I've found the setting for 'terminate other sessions' but it's unchecked so not sure where else to look.
Could someone help with this please?
